extensions/typescript-language-features/src/tsServer/spawner.ts

```
@@ -216,6 +226,13 @@ export class TypeScriptServerSpawner {
 			}
 		}
 
+		if (configuration.enableTsServerTracing && !isWeb()) {
```

Does this need a version check?

---

I thought the server would silently ignore unsupported arguments.